src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -411,13 +411,18 @@ export class CellExecution {
             return this.completedSuccessfully().then(noop, noop);
         }
 
+        // For Jupyter requests, silent === don't output, while store_history === don't update execution count
+        // https://jupyter-client.readthedocs.io/en/stable/api/client.html#jupyter_client.KernelClient.execute
+        // for our usage (such as variable requests) we do want output, but we don't want to update execution count
+        const executeSilently = false;
```

Comment:
I added this here as a constant. Basically I added this to keep separate our concept of execute silent (silent=false, store_history=false) from the jupyter concept which is silent = don't show output.

Reply:
Not a fan of negating another variable.
silent & store_history are two seprate flags.
However to we're not usign a single flag and negating it to use in the place of `store_history`. Feels confusing when we do that.
I.e. Why not have `storeHistory` as a variable, instead of creating `silent`. after all, Jupyter has two separate variables for that, why are we try to merge them into one & assume that store_history is the opposite of `silent` (that's the impression i get from this code).

Reply:
I'd prefer to have two separate constants.

Reply:
@DonJayamanne Hmmm, taking another look. I don't think that we need this here at all. I was adding it as I considered it a bit confusing as we have our own concept of silent in our code above the JMP level. Basically what we call silent corresponds to store_history, in all our old code we never actually set silent, we just translate silent to store_history at the execution level since we always wanted output. But since it's hard coded here I can just ditch the constant and have the logger base silent off of the store_history value of the request. That might work better if the constant was just introducing more confusion.

Reply:
Got it, thanks for clarification.